src/api/custom-schema/type-defs.ts

```diff
@@ -235,11 +235,28 @@ const typeDefs = `
   }
 
   type Owner {
+    id: String
     firstName: String
     lastName: String
     email: String
   }
 
+  type FinancialUk {
```

Comment:
`ApplicationLossPayeeFinancialUk`?

Reply:
Has this been changed? Not showing for me

Reply:
updated to ApplicationNominatedLossPayeeUk